README.md

```diff
@@ -37,6 +37,12 @@ To use the Django Girls name for your workshop, your event needs to meet the fol
 * it has to be non-profit
 * it has to be approved by the Django Girls Foundation, remember to contact us first! Details below.
 
+To ensure the quality of each Django Girls workshop, and to support local infrastructure, volunteers and supporters from becoming overextended:
+* applications should be placed a minimum of __three months__ prior to the proposed workshop date to allow adequate time for planning
+* the proposed workshop date should be at least 6 months after the previous workshop in that region
+* applications should be made a minimum of 4 weeks following the previous workshop in that region, to ensure there is sufficient time to complete any follow up tasks before planning for the next workshop begins
+* *in unique circumstances, we may allow exemptions from the above first two rules. Please contact "hello@djangogirls.org" and let us know why running the event in the suggested timeframe is not an option for you.
```

Comment:
Do we need a `*` starting this line or we wanted to bold `in unique circumstances`?